SimplyPluralApiBodies.yml

```diff
@@ -4,30 +4,68 @@ info:
   description: This API reference documents the public api endpoints. You can see a list of all public endpoints here https://github.com/ApparyllisOrg/SimplyPluralApi/blob/release/src/api/v1/routes.ts
   version: "1.0"
 
-paths:
-  /somePath:
-    get:
-      responses:
-        200:
-          description: ok
-
 components:
   schemas:
-    commons:
+    analytics:
       type: object
       properties:
-        color:
-          description: Color of the object (\#ffffff, \#ffffffff or ffffff)
-        private:
-          description: Whether this object is private
-        preventTrusted:
-          description: Wheter trusted friends are prevented from seeing this private object
-        supportDescMarkdown:
-          description: Whether the description supports markdown
-        timestamp:
-          description: Milliseconds since epoch for this timestamp/date
+        timings:
+          description: Groups of fronters based on timings
+          type: object
+          properties:
+            morningFronters:
+              type: array
+              items:
+                type: string
+            dayFronters:
+              type: array
+              items:
+                type: string
+            eveningFronters:
+              type: array
+              items:
+                type: string
+            nightFronters:
+              type: array
+              items:
+                type: string
+        values:
+          description: Values for the timeframe
+          type: object
+          properties:
+            sums:
+              type: array
+              items:
+                $ref: "#/components/schemas/analyticsIdValuePair"
+            averages:
+              type: array
+              items:
+                $ref: "#/components/schemas/analyticsIdValuePair"
+            maxes:
+              type: array              
+              items:
+                $ref: "#/components/schemas/analyticsIdValuePair"
+            mins:
+              type: array
+              items:
+                $ref: "#/components/schemas/analyticsIdValuePair"
+            nums:
+              type: array
+              items:
+                $ref: "#/components/schemas/analyticsIdValuePair"
+
+    analyticsIdValuePair:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The Id of the member
+        value:
+          type: long
+          description:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/timestamp/description"
```

Comment:
This is not correct, the id pair id is correct, but the value differs per result.

Sums : Value => Total length they fronted 
Averages: Value => Average length they fronted
Maxes: Value => Maximum length they fronted
Mins: Value => Minimum length they fronted
Nums: Value => Amount of times they fronted